scripts/notifications.coffee

```diff
@@ -213,4 +252,30 @@ e.g. --message Hello World --channel general tech --> "Hello World" used as --me
       # Add necessary configurations to vars
       vars.plugins[medium] = varsTemp[medium]
 
-    doRequest(vars, res)
+    # Verify message:
+    robot.messageRoom CONTROL_CHANNEL_NAME, "New notification pending. How's it look? (type `botgt notify approve` or `botgt notify abort`)"
```

Comment:
Can we check if there's a already pending message (by my understanding, this code would overwrite the previous pending message with the new pending message)

Reply:
What action do you want? Bounce the current one? I thought overwrite would be a could default behavior.